go/integTest/standalone_commands_test.go

```diff
@@ -44,6 +45,32 @@ func (suite *GlideTestSuite) TestCustomCommandClientInfo() {
 	assert.True(suite.T(), strings.Contains(strResult, fmt.Sprintf("name=%s", clientName)))
 }
 
+func (suite *GlideTestSuite) TestCustomCommandMGET() {
+	clientName := "TEST_CLIENT_NAME"
+	config := api.NewGlideClientConfiguration().
+		WithAddress(&api.NodeAddress{Port: suite.standalonePorts[0]}).
+		WithClientName(clientName)
+	client := suite.client(config)
+
+	key1 := uuid.New().String()
+	key2 := uuid.New().String()
+	key3 := uuid.New().String()
+	oldValue := uuid.New().String()
+	value := uuid.New().String()
+	suite.verifyOK(client.Set(key1, oldValue))
+	keyValueMap := map[string]string{
+		key1: value,
+		key2: value,
+	}
+	suite.verifyOK(client.MSet(keyValueMap))
+	keys := []string{key1, key2, key3}
+	values := []interface{}{value, value, nil}
+	result, err := client.CustomCommand(append([]string{"MGET"}, keys...))
```

Comment:
Try to test with custom command with different return types, for example `config get` returns a map, `del` returns an integer

Reply:
Added the tests for other types as well.